src/bob/config.clj

```diff
@@ -0,0 +1,76 @@
+;   This file is part of Bob.
+;
+;   Bob is free software: you can redistribute it and/or modify
+;   it under the terms of the GNU Affero General Public License as published by
+;   the Free Software Foundation, either version 3 of the License, or
+;   (at your option) any later version.
+;
+;   Bob is distributed in the hope that it will be useful,
+;   but WITHOUT ANY WARRANTY; without even the implied warranty of
+;   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+;   GNU Affero General Public License for more details.
+;
+;   You should have received a copy of the GNU Affero General Public License
+;   along with Bob. If not, see <http://www.gnu.org/licenses/>.
+
+(ns bob.config
+  (:require [clojure.edn :as edn]
+            [environ.core :as env :refer [env]]
+            [schema.core :as s]
+            [taoensso.timbre :as log]
+            [bob.api.schemas :as schema]))
+
+(defn int-from-env
+  [key default]
+  (try
+    (Integer/parseInt (get env key (str default)))
+    (catch Exception _ default)))
+
+(defn read-config-file
+  "read config from .bob.conf"
+  []
+  (try
+    (when-let [env-file (clojure.java.io/file "./.bob.conf")]
```

Comment:
I think it could be better to read this from `~/.bob.conf` than the root folder as it would be difficult to know were the JAR is or the PWD depending on the method of deployment.

Reply:
would this be working for windows as well? Should I use `Paths.get(System.getProperty("user.home"))` for platform independence?

Reply:
Yes we need to. Also I dont thing java understands ~ either, so we have more reasons.

Reply:
done.  `(str (System/getProperty "user.home") "/.bob.conf")`